CONSTRAINTS.md

```diff
@@ -13,18 +13,21 @@ One of main tensions with designing the CLE is providing enough structure to mak
   - Prevalence - How universally they apply across different component types and ecosystems.
   - Impact - How much they influence risk and management of a component.
 - The events SHOULD enable component consumers to make informed decisions about risk, supportability, and upgrades.
-- The events MUST be unambiguous and factual in nature, not speculative or subjective.
+- The events MUST be unambiguous and factual in nature, avoiding speculative or subjective claims. For instance, an event must not rely on interpretations such as "we think this project by this maintainer is dead". Further guidance on claims is provided below.
```

Comment:
This still doesn’t account for future events - iow, we should encourage maintainers to decide EOL dates in advance, and communicate them - but these are inherently speculative in nature.

Reply:
@ljharb let me know what you think of this updated wording


> The events MUST be unambiguous and factual in nature, and they MUST NOT include speculative or subjective claims. An event MUST NOT rely on speculative inference, such as "we think this project X by this maintainer Y is dead because of the absence of commits", which is based on indirect evidence or interpretation. Events MUST be based on direct evidence, such as a maintainer announcing the end of a project, or a project being archived. Further guidance on claims is provided below.